index.html

```diff
@@ -2520,6 +2519,19 @@ <h3><dfn>Writing content</dfn></h3>
                 If |source|'s records [= list/is empty =], [= exception/throw =]
                 a {{TypeError}} and abort these steps.
               </li>
+              <li>
+                Increment |recordsDepth| by one.
+              </li>
+              <li>
+                If |recordsDepth| > `32`, [= exception/throw =] a {{TypeError}}
+                and abort these steps.
+                <p class="issue" data-number="620">
```

Comment:
Should we be using `class="issue"` here? Merging this change will resolve the issue so this should probably be a regular note, with reference to the issue that caused the change.